mesonbuild/backend/vs2010backend.py

```diff
@@ -1192,7 +1192,8 @@ def gen_vcxproj(self, target, ofname, guid):
         # /nologo
         ET.SubElement(link, 'SuppressStartupBanner').text = 'true'
         # /release
-        ET.SubElement(link, 'SetChecksum').text = 'true'
+        if (self.buildtype not in ['debug', 'debugoptimized']):
+            ET.SubElement(link, 'SetChecksum').text = 'true'
```

Comment:
Is this "debug" checking for debug info or for debug CRT?

If the former, this should check for ` debug` and `b_vscrt` if the latter.

Reply:
its checking for debug info

Reply:
In that case the code should check for `self.environment.coredata.get_builtin_option('debug')` instead of the build type.